ngx-tools/type-guards/README.md

```diff
@@ -288,3 +288,17 @@ isUndefined(undefined) // Returns: true
 isUndefined(null)      // Returns: false
 isUndefined('foo')     // Returns: false
 ```
+
+## `isValidDate`
```

Comment:
hmm seems the table of contents didn't get updated.. maybe we didn't migrate all the UI library git hooks over to the tools lib?